src/Components/ListArticles.js

```diff
@@ -28,10 +34,14 @@ class ListArticles extends Component {
     let articleList
     let errorMessage
 
+    if (this.props.getArticlesIndexProps === true) {
```

Comment:
Don't need this anymore, right? Please revert component back to upstream development branch state.